storage/postgres/client.go

```diff
@@ -309,6 +309,10 @@ func (c *Client) listIndexerMaterializedViews(ctx context.Context) ([]string, er
 
 // Wipe removes all contents of the database.
 func (c *Client) Wipe(ctx context.Context) error {
+	if _, err := c.pool.Exec(ctx, "DROP EXTENSION IF EXISTS pg_trgm CASCADE;"); err != nil {
```

Comment:
this works out okay when it's at the beginning, because this way dropping the extension with cascade cleans up its types and functions